src/vars.js

```diff
@@ -0,0 +1,4 @@
+const DEFAULT_PAGE = 0;
+const DEFAULT_PER_PAGE = 20;
```

Comment:
Nice. This is the same approach you should use for strings that get re-used (e.g. book title exists)